paasta_tools/setup_marathon_job.py

```diff
@@ -392,6 +399,17 @@ def log_bounce_action(line: str, level: str='debug') -> None:
                 ),
                 level='event',
             )
+
+            if yelp_meteorite:
+                # Emit deploy.paasta event
+                yelp_meteorite.events.emit_event(
+                    'deploy.paasta',
+                    dimensions={
+                        'cluster': cluster,
```

Comment:
Here is some bad news: we haven't been consistent in our dimensions with our metrics.
https://github.com/Yelp/paasta/blob/master/paasta_tools/contrib/emit_allocated_cpu_metrics.py#L26

What do you think about getting us off onto the "right" foot and moving us twoards a consistent `paasta_cluster`, `paasta_service` and `paasta_instance` ?

Reply:
I'll go ahead and rename these dimensions to be consistent with others.